config/queryparam-extensions.json

```diff
@@ -1,12 +1,51 @@
 {
   "Patient": {
     "mothersMaidenName.given": {
+      "extension": true,
       "url": "http://pdqm-sample:8080/ITI-78/Profile/pdqm#mothersMaidenName",
-      "valuePropPath": "valueHumanName.given"
+      "path": "valueHumanName.given",
+      "type": "string",
+      "propertyDef": {
+        "min": 0,
+        "max": "1",
```

Comment:
Im wondering if we should allow multiple given names? This then keeps it inline with the HumanName type